src/test/java/com/coupang/marketplace/coupon/service/CouponServiceTest.java

```diff
@@ -0,0 +1,38 @@
+package com.coupang.marketplace.coupon.service;
+
+import static com.coupang.marketplace.fixture.CouponFixture.Coupon1.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.coupang.marketplace.coupon.repository.CouponRepository;
+import com.coupang.marketplace.fixture.CouponFixture;
+
+@ExtendWith(MockitoExtension.class)
+public class CouponServiceTest {
+
+	@InjectMocks
+	private CouponService couponService;
+
+	@Mock
+	private CouponRepository couponRepository;
+
+	@DisplayName("만료시간이 지나지 않은 쿠폰을 모두 가져온다.")
+	@Test
+	public void getCouponsBeforeExpirationTime() {
+		//given
+		given(couponRepository.getCouponsBeforeExpirationTime())
+			.willReturn(Arrays.asList(COUPON));
```

Comment:
```suggestion
			.willReturn(Arrays.asList(Coupon1.COUPON));
```
이렇게 쓰면 어떤 fixture 를 생성한 것인지 명확해질 것 같습니다!

Reply:
그렇겠네요~ 수정하겠습니다🤗